niri-ipc/src/lib.rs

```diff
@@ -996,6 +996,19 @@ pub struct Window {
     ///
     /// If the window isn't floating then it is in the tiling layout.
     pub is_floating: bool,
+    /// Location of the window within a workspace in terms of (column index, tile index in column).
+    ///
+    /// Unset for floating windows.
+    pub tile_coordinates: Option<(usize, usize)>,
```

Comment:
```suggestion
    pub tile_pos_in_scrolling_layout: Option<(usize, usize)>,
```
Idk? Kind of a mouthful still but I want to communicate this.